src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java

```diff
@@ -413,19 +408,21 @@ public int compare(Pair<Coordinate, Double> coordinateDoublePair, Pair<Coordinat
         log.trace("worst Point List:" + worstEdgePoints.size() + "first element: " + worstEdgePoints.get(0));
         log.debug("best overall Sampled constant is" + bestSampled.getValue() + " at " + bestSampled.getKey().toString());
         log.debug("best overall calculated constant is" + bestCalc.getValue() + " at " + bestCalc.getKey().toString());
-        if(PreferenceService.getInstance().getPreference(MobileTreasureHider.walkedPathLengthForTreasureRelocation_Preference, 1).intValue() == 1){
+        if (PreferenceService.getInstance().getPreference(MobileTreasureHider.walkedPathLengthForTreasureRelocation_Preference, 1).intValue() == 1) {
             return worstEdgePointsSampled;
-        }else{
+        } else {
             return worstEdgePoints;
         }
     }
 
 
-    /**Todo
+    /**
+     * Todo
```

Comment:
No

Reply:
Big nono

Reply:
The author of this file should add correct documentation. @Trampeltier @rukemn @DorianRec ?

Reply:
this were also build by autoformatting ..
I agree, but Im not the author ^^